{"name": "test_correct_redirection_after_valid_email_submitted_TC_009", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Element for xpath: '//*[@id=\"account-creation_form\"]'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found", "trace": "self = <tests.pages.test_authentication_page.AuthenticationPageSanityTests testMethod=test_correct_redirection_after_valid_email_submitted_TC_009>\n\n    def test_correct_redirection_after_valid_email_submitted_TC_009(self):\n        self.authentication_page.visit()\n        self.authentication_page.input_data_and_click(AuthLocators.email_input_field_xpath, Input.valid_email,\n                                                      AuthLocators.create_account_button)\n>       self.visibility_of_element_wait(self.driver, CreateLocators.create_account_form_xpath)\n\ntests/pages/test_authentication_page.py:59: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/pages/base_test_class.py:66: in visibility_of_element_wait\n    return wait.until(element_located, f\"Element for xpath: '{xpath}'and url: {driver.current_url} not found\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"88773b0aa367696bf2a8e169ac390344\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x7f75c6f98eb0>\nmessage = 'Element for xpath: \\'//*[@id=\"account-creation_form\"]\\'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found'\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Element for xpath: '//*[@id=\"account-creation_form\"]'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found\n\nvenv/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "065057be-11d1-40ee-a68f-16a8b67b8537-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "ed99a528-2e33-46f5-8f70-4a280166b4c7-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "719f167b-c88c-4150-91c9-a1142269ee74-attachment.txt", "type": "text/plain"}], "start": 1622592097111, "stop": 1622592115265, "uuid": "5126b297-b781-4ded-88a8-f00163afd91a", "historyId": "c876d739becb5f872cb75fd631b0440f", "testCaseId": "66a69d5fd49632595a299b1a693f4235", "fullName": "tests.test_suites.testsuite_all_tests.AuthenticationPageSanityTests#test_correct_redirection_after_valid_email_submitted_TC_009", "labels": [{"name": "parentSuite", "value": "tests.test_suites"}, {"name": "suite", "value": "testsuite_all_tests"}, {"name": "subSuite", "value": "AuthenticationPageSanityTests"}, {"name": "host", "value": "pehaLin"}, {"name": "thread", "value": "92448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_suites.testsuite_all_tests"}]}